### ISSUE TITLE
For Ubuntu 24.04 (noble) Debian 64-bit time_t Transition package with "t64" in name like libglib2.0-0t64 are not resolved in Aptly 1.5.0 

### ISSUE THREAD

**Issue by BrightGum** [aptly-dev/aptly#1318]
It seem that with Aptly 1.5.0 for Ubuntu 24.04 (noble) Debian 64-bit time_t transition packages with "t64" in name like libglib2.0-0t64 are not resolved in _search_ or _filter_.

How to reproduce:

- Create a mirror of Ubuntu 24.04 (noble) and download the repositories:

```

aptly mirror create \
  -filter="" \
  -dep-follow-all-variants \
  -dep-follow-recommends \
  -dep-follow-suggests \
  -dep-verbose-resolve \
  -architectures=amd64 noble https://ftp.fau.de/ubuntu/ noble

aptly mirror create \
  -filter="" \
  -dep-follow-all-variants \
  -dep-follow-recommends \
  -dep-follow-suggests \
  -dep-verbose-resolve \
  -architectures=amd64 noble-security https://ftp.fau.de/ubuntu/ noble-security

aptly mirror create \
  -filter="" \
  -dep-follow-all-variants \
  -dep-follow-recommends \
  -dep-follow-suggests \
  -dep-verbose-resolve \
  -architectures=amd64 noble-updates https://ftp.fau.de/ubuntu/ noble-updates

aptly mirror create \
  -filter="" \
  -dep-follow-all-variants \
  -dep-follow-recommends \
  -dep-follow-suggests \
  -dep-verbose-resolve \
  -architectures=amd64 noble-backports https://ftp.fau.de/ubuntu/ noble-backports

aptly mirror update noble
aptly mirror update noble-updates
aptly mirror update noble-security
aptly mirror update noble-backports

```
- Just a mirror list as overview:
```
root@aptly-2404-v2:~# aptly mirror list 
List of mirrors:
 * [noble-backports]: https://ftp.fau.de/ubuntu/ noble-backports
 * [noble-security]: https://ftp.fau.de/ubuntu/ noble-security
 * [noble-updates]: https://ftp.fau.de/ubuntu/ noble-updates
 * [noble]: https://ftp.fau.de/ubuntu/ noble

```

- Create snapshots of the mirrors:
```
aptly snapshot create noble-snapshot-2024-07-17-151128 from mirror noble
aptly snapshot create noble-updates-snapshot-2024-07-17-151128 from mirror noble-updates
aptly snapshot create noble-backports-snapshot-2024-07-17-151128 from mirror noble-backports
aptly snapshot create noble-security-snapshot-2024-07-17-151128 from mirror noble-security


```

- Merge all noble (noble, updates, backports, security) snapshots to one:
```
aptly snapshot merge -no-remove \
 test-merge-1 \
 noble-snapshot-2024-07-17-151128 \
 noble-updates-snapshot-2024-07-17-151128 \
 noble-backports-snapshot-2024-07-17-151128 \
 noble-security-snapshot-2024-07-17-151128 \

```

- To reproduce, we use the following example _'Name (= libdconf1)'_
```

aptly snapshot \
 -dep-verbose-resolve \
 -with-deps \
 -dep-follow-all-variants \
 -dep-follow-recommends \
 search test-merge-1 \
 'Name (= libdconf1)'

Missing dependencies: libc6 (>= 2.4) [amd64], libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: libc6_2.39-0ubuntu8.2_amd64
Injecting package: libc6_2.39-0ubuntu8_amd64
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
Missing dependencies: libgcc-s1 [amd64], libidn2-0 (>= 2.0.5~) [amd64], libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: libgcc-s1_14-20240412-0ubuntu1_amd64
Injecting package: libidn2-0_2.3.7-2build1_amd64
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
Missing dependencies: libglib2.0-0 (>= 2.79.0) [amd64], gcc-14-base (= 14-20240412-0ubuntu1) [amd64], libunistring5 (>= 1.1) [amd64]
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: gcc-14-base_14-20240412-0ubuntu1_amd64
Injecting package: libunistring5_1.1-2build1_amd64
Missing dependencies: libglib2.0-0 (>= 2.79.0) [amd64]
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
gcc-14-base_14-20240412-0ubuntu1_amd64
libc6_2.39-0ubuntu8.2_amd64
libc6_2.39-0ubuntu8_amd64
libdconf1_0.40.0-4build2_amd64
libgcc-s1_14-20240412-0ubuntu1_amd64
libidn2-0_2.3.7-2build1_amd64
libunistring5_1.1-2build1_amd64

```

- We see the _Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]_

```

aptly snapshot \
 -dep-verbose-resolve \
 -with-deps \
 search test-merge-1 \
 'Name (= libglib2.0-0)'
ERROR: no results

```

- As we can see there are some _libglib2.0-0t64__
```

root@aptly-2404-v2:~# aptly snapshot search test-merge-1  'Name (~ libglib2.0-0)'
libglib2.0-0t64_2.80.0-6ubuntu3.1_amd64
libglib2.0-0t64_2.80.0-6ubuntu1_amd64

```

- On a Ubuntu 24.04 with the standard online repositories we see the following with _apt_:
```

root@aptly-2404-v2:~# export LC_ALL=C && apt search libglib2.0-0
Sorting... Done
Full Text Search... Done
libglib2.0-0t64/noble-updates,noble-security,now 2.80.0-6ubuntu3.1 amd64 [installed,automatic]
  GLib library of C routines

libglib2.0-dev/noble-updates,noble-security 2.80.0-6ubuntu3.1 amd64
  Development files for the GLib library

libglib2.0-dev-bin/noble-updates,noble-security 2.80.0-6ubuntu3.1 amd64
  Development utilities for the GLib library



```
As background information:
We need an offline Ubuntu 24.04 repository with minimal footprint. In addition to the noble repositories, there are others. Therefore I add an extended example, which creates a non-functioning repository  _test-merge-2_ .

```
aptly snapshot  -dep-verbose-resolve  -with-deps  search test-merge-2  'Name (= accountsservice)|Name (= acl)|Name (= aksusbd)|Name (= alsa-ucm-conf)|Name (= apache2)|Name (= apport)|Name (= apt)|Name (= apt-utils)|Name (= archivemount)|Name (= at)|Name (= attr)|Name (= base-files)|Name (= base-passwd)|Name (= bash)|Name (= bash-builtins)|Name (= bash-completion)|Name (= bash-doc)|Name (= bash-static)|Name (= bc)|Name (= bcache-tools)|Name (= bind9-dnsutils)|Name (= bind9-host)|Name (= bind9-libs)|Name (= bmon)|Name (= build-essential)|Name (= busybox-initramfs)|Name (= busybox-static)|Name (= byobu)|Name (= ca-certificates)|Name (= casper)|Name (= ceph)|Name (= ceph-base)|Name (= ceph-common)|Name (= ceph-fuse)|Name (= chef)|Name (= cifs-utils)|Name (= clientmate)|Name (= cloud-guest-utils)|Name (= cloud-init)|Name (= cloud-initramfs-copymods)|Name (= cloud-initramfs-dyn-netconf)|Name (= colord)|Name (= command-not-found)|Name (= console-setup)|Name (= console-setup-linux)|Name (= corosync)|Name (= cpio)|Name (= cpufrequtils)|Name (= crmsh)|Name (= cron)|Name (= ctop)|Name (= cups-bsd)|Name (= cups-filters)|Name (= curl)|Name (= cvs)|Name (= dash)|Name (= dbus)|Name (= dbus-user-session)|Name (= ddd)|Name (= debootstrap)|Name (= dialog)|Name (= diffutils)|Name (= dirmngr)|Name (= distro-info-data)|Name (= dmidecode)|Name (= dnsmasq)|Name (= dnsutils)|Name (= docker-compose)|Name (= docker.io)|Name (= dos2unix)|Name (= dosfstools)|Name (= dpkg-dev)|Name (= duf)|Name (= e2fslibs)|Name (= eatmydata)|Name (= ed)|Name (= efibootmgr)|Name (= errepiusv)|Name (= ethtool)|Name (= exa)|Name (= expect)|Name (= fakeroot)|Name (= fdisk)|Name (= findutils)|Name (= firefox)|Name (= fonts-ubuntu-console)|Name (= freeipmi-tools)|Name (= friendly-recovery)|Name (= ftp)|Name (= fusesmb)|Name (= fwupd)|Name (= fwupd-signed)|Name (= gdisk)|Name (= gettext)|Name (= git)|Name (= git-man)|Name (= glib-networking)|Name (= glib-networking-common)|Name (= glib-networking-services)|Name (= gnupg)|Name (= gnupg-l10n)|Name (= gnupg-utils)|Name (= gpg)|Name (= gpg-agent)|Name (= gpgconf)|Name (= gpgsm)|Name (= gpgv)|Name (= gpg-wks-client)|Name (= gpg-wks-server)|Name (= graphviz)|Name (= grep)|Name (= grub-efi-amd64)|Name (= grub-efi-amd64-signed)|Name (= gzip)|Name (= hdparm)|Name (= hostname)|Name (= htop)|Name (= ifenslave)|Name (= ifstat)|Name (= iftop)|Name (= imagemagick)|Name (= imvirt)|Name (= info)|Name (= init)|Name (= initramfs-tools)|Name (= initramfs-tools-bin)|Name (= initramfs-tools-core)|Name (= iotop)|Name (= iperf)|Name (= ipmitool)|Name (= ipp-linux)|Name (= iptraf)|Name (= iputils-ping)|Name (= iputils-tracepath)|Name (= irqbalance)|Name (= isc-dhcp-client)|Name (= isc-dhcp-common)|Name (= joe)|Name (= krb5-kdc)|Name (= krb5-locales)|Name (= krb5-user)|Name (= landscape-common)|Name (= ldap-utils)|Name (= ledmon)|Name (= less)|Name (= lib32z1)|Name (= libaccountsservice0)|Name (= libalgorithm-diff-perl)|Name (= libalgorithm-diff-xs-perl)|Name (= libalgorithm-merge-perl)|Name (= libapache2-mod-php)|Name (= libapt-pkg6.0)|Name (= libasound2)|Name (= libasound2-data)|Name (= libauthen-sasl-perl)|Name (= libblkid1)|Name (= libbrotli1)|Name (= libc-bin)|Name (= libcryptsetup12)|Name (= libcurl3-gnutls)|Name (= libdebconfclient0)|Name (= libdigest-hmac-perl)|Name (= libdns-export1109)|Name (= libdrm2)|Name (= libdrm-common)|Name (= libeatmydata1)|Name (= libefiboot1)|Name (= libefivar1)|Name (= libfdisk1)|Name (= libfile-fcntllock-perl)|Name (= libfl2)|Name (= libfont-afm-perl)|Name (= libfwupd2)|Name (= libfwupdplugin1)|Name (= libgcc1)|Name (= libgcrypt20)|Name (= libglib2.0-0)|Name (= libglib2.0-bin)|Name (= libglib2.0-data)|Name (= libgnutls30)|Name (= libgssapi-krb5-2)|Name (= libgssapi-perl)|Name (= libhogweed5)|Name (= libhtml-format-perl)|Name (= libhtml-form-perl)|Name (= libhttp-daemon-perl)|Name (= libicu66)|Name (= libisc-export1105)|Name (= libisns0)|Name (= libk5crypto3)|Name (= libkrb5-3)|Name (= libkrb5support0)|Name (= libldap-common)|Name (= liblz4-1)|Name (= liblzma5)|Name (= libmailtools-perl)|Name (= libmaxminddb0)|Name (= libmount1)|Name (= libmspack0)|Name (= libncurses-dev)|Name (= libnetplan0)|Name (= libnettle7)|Name (= libnss-winbind)|Name (= libnuma1)|Name (= libnuma-dev)|Name (= libp11-kit0)|Name (= libpam0g)|Name (= libpam-modules)|Name (= libpam-modules-bin)|Name (= libpam-runtime)|Name (= libpam-winbind)|Name (= libpci3)|Name (= libperl5.30)|Name (= libprocps8)|Name (= libproxy1v5)|Name (= libpython3.8)|Name (= libpython3.8-minimal)|Name (= libpython3.8-stdlib)|Name (= libseccomp2)|Name (= libsmartcols1)|Name (= libsodium23)|Name (= libssh-4)|Name (= libssl1.1)|Name (= libterm-readkey-perl)|Name (= libudev1)|Name (= liburcu6)|Name (= libuuid1)|Name (= libuv1)|Name (= libwww-perl)|Name (= libx11-6)|Name (= libx11-data)|Name (= libxml2)|Name (= libxml2-utils)|Name (= libxmlsec1)|Name (= libxmlsec1-openssl)|Name (= libxml-xpath-perl)|Name (= libzstd1)|Name (= linux-base)|Name (= linux-generic)|Name (= linux-generic-hwe-20.04)|Name (= linux-hwe-5.15-tools-common)|Name (= linux-image-generic-hwe-20.04)|Name (= linux-tools-common)|Name (= linux-tools-generic)|Name (= linux-tools-generic-hwe-20.04)|Name (= locales)|Name (= login)|Name (= lsb)|Name (= lsb-core)|Name (= lshw)|Name (= lsof)|Name (= lsscsi)|Name (= ltrace)|Name (= lupin-casper)|Name (= lxd-agent-loader)|Name (= lz4)|Name (= man-db)|Name (= mawk)|Name (= mdadm)|Name (= memtest86+)|Name (= mmdebstrap)|Name (= mongodb)|Name (= monit)|Name (= mount)|Name (= mtools)|Name (= mtr-tiny)|Name (= multipath-tools)|Name (= nano)|Name (= ncurses-base)|Name (= ncurses-bin)|Name (= ndiff)|Name (= nedit)|Name (= netplan.io)|Name (= networkd-dispatcher)|Name (= network-manager)|Name (= nfs-kernel-server)|Name (= nicstat)|Name (= nmap)|Name (= nmon)|Name (= node_exporter)|Name (= nscd)|Name (= ntfs-3g)|Name (= ntp)|Name (= ntpdate)|Name (= numactl)|Name (= open-iscsi)|Name (= openjdk-17-jdk)|Name (= openjdk-17-jre)|Name (= openjdk-8-jdk)|Name (= openjdk-8-jre)|Name (= openssh-server)|Name (= openssl)|Name (= open-vm-tools)|Name (= overlayroot)|Name (= p7zip)|Name (= p7zip-full)|Name (= pacemaker)|Name (= parallel)|Name (= parted)|Name (= passwd)|Name (= pastebinit)|Name (= pbzip2)|Name (= pciutils)|Name (= perl)|Name (= perl-base)|Name (= perl-modules-5.30)|Name (= pollinate)|Name (= popularity-contest)|Name (= postgresql-13)|Name (= postgresql-13-postgis-3)|Name (= powermgmt-base)|Name (= procps)|Name (= prometheus)|Name (= prometheus-alertmanager)|Name (= prometheus-apache-exporter)|Name (= prometheus-bind-exporter)|Name (= prometheus-bird-exporter)|Name (= prometheus-blackbox-exporter)|Name (= prometheus-elasticsearch-exporter)|Name (= prometheus-exporter-exporter)|Name (= prometheus-frr-exporter)|Name (= prometheus-hacluster-exporter)|Name (= prometheus-haproxy-exporter)|Name (= prometheus-homeplug-exporter)|Name (= prometheus-ipmi-exporter)|Name (= prometheus-libvirt-exporter)|Name (= prometheus-mailexporter)|Name (= prometheus-mongodb-exporter)|Name (= prometheus-mqtt-exporter)|Name (= prometheus-mysqld-exporter)|Name (= prometheus-nextcloud-exporter)|Name (= prometheus-nginx-exporter)|Name (= prometheus-node-exporter)|Name (= prometheus-node-exporter-collectors)|Name (= prometheus-openstack-exporter)|Name (= prometheus-pgbouncer-exporter)|Name (= prometheus-postfix-exporter)|Name (= prometheus-postgres-exporter)|Name (= prometheus-process-exporter)|Name (= prometheus-pushgateway)|Name (= prometheus-redis-exporter)|Name (= prometheus-smokeping-prober)|Name (= prometheus-snmp-exporter)|Name (= prometheus-sql-exporter)|Name (= prometheus-squid-exporter)|Name (= prometheus-tplink-plug-exporter)|Name (= prometheus-trafficserver-exporter)|Name (= prometheus-varnish-exporter)|Name (= prometheus-xmpp-alerts)|Name (= psmisc)|Name (= pxelinux)|Name (= python2)|Name (= python3)|Name (= python3.8)|Name (= python3.8-minimal)|Name (= python3-apport)|Name (= python3-cffi-backend)|Name (= python3-commandnotfound)|Name (= python3-debian)|Name (= python3-distupgrade)|Name (= python3-distutils)|Name (= python3-gdbm)|Name (= python3-jinja2)|Name (= python3-jsonpatch)|Name (= python3-json-pointer)|Name (= python3-lib2to3)|Name (= python3-nacl)|Name (= python3-newt)|Name (= python3-paramiko)|Name (= python3-pip)|Name (= python3-problem-report)|Name (= python3-psycopg2)|Name (= python3-pymacaroons)|Name (= python3-ruamel.yaml)|Name (= python3-serial)|Name (= python3-setuptools)|Name (= python3-smt-api)|Name (= python3-software-properties)|Name (= python3-software-properties-common)|Name (= python3-update-manager)|Name (= python3-wheel)|Name (= python3-yaml)|Name (= python-apt-common)|Name (= python-is-python2)|Name (= qemu-guest-agent)|Name (= qpdf)|Name (= rar)|Name (= rcs)|Name (= rsyslog)|Name (= rsyslog-relp)|Name (= run-one)|Name (= saab-admin-scripts)|Name (= samba)|Name (= sambaclient)|Name (= samba-common-bin)|Name (= samba-vfs-modules)|Name (= screen)|Name (= sendemail)|Name (= sg3-utils)|Name (= sg3-utils-udev)|Name (= shim-signed)|Name (= smartmontools)|Name (= smbclient)|Name (= snapd)|Name (= snmp)|Name (= snmpd)|Name (= snmptrapd)|Name (= snmptt)|Name (= socat)|Name (= software-properties-common)|Name (= sosreport)|Name (= sqlite3)|Name (= sssd)|Name (= sssd-ad)|Name (= sssd-ad-common)|Name (= sssd-common)|Name (= sssd-dbus)|Name (= sssd-ipa)|Name (= sssd-kcm)|Name (= sssd-krb5)|Name (= sssd-krb5-common)|Name (= sssd-ldap)|Name (= sssd-proxy)|Name (= sssd-tools)|Name (= strace)|Name (= sudo)|Name (= sysbench)|Name (= syslinux)|Name (= syslinux-common)|Name (= syslinux-utils)|Name (= sysstat)|Name (= sysvinit-utils)|Name (= tasksel)|Name (= tcpd)|Name (= tcpdump)|Name (= telnet)|Name (= time)|Name (= tmux)|Name (= traceroute)|Name (= tzdata)|Name (= ubuntu-advantage-tools)|Name (= ubuntu-keyring)|Name (= ubuntu-minimal)|Name (= ubuntu-release-upgrader-core)|Name (= ubuntu-release-upgrader-qt)|Name (= ubuntu-server)|Name (= ubuntu-standard)|Name (= udev)|Name (= ufw)|Name (= unrar)|Name (= unzip)|Name (= update-manager-core)|Name (= update-notifier-common)|Name (= util-linux)|Name (= vim)|Name (= vlan)|Name (= winbind)|Name (= xauth)|Name (= xdg-user-dirs)|Name (= xfsprogs)|Name (= xgrep)|Name (= xinetd)|Name (= xml-core)|Name (= xmlstarlet)|Name (= xorg)|Name (= xorriso)|Name (= xqilla)|Name (= xserver-xorg)|Name (= xterm)|Name (= xvfb)|Name (= xxd)|Name (= xz-utils)|Name (= zerofree)|Name (= zfsutils-linux)|Name (= zfs-zed)|Name (= zip)' 
```

**Comment by BrightGum** [comment]
I have repeated the test with version 1.5.0+172+g57e2c5c6 and get the same result:
```
root@aptly-2404-v2:~/git/aptly# /root/go/bin/aptly version
aptly version: 1.5.0+172+g57e2c5c6
```

```
root@aptly-2404-v2:~/git/aptly# aptly snapshot \
>  -dep-verbose-resolve \
>  -with-deps \
>  -dep-follow-all-variants \
>  -dep-follow-recommends \
>  search test-merge-2024-07-25-100000 \
>  'Name (= libdconf1)'
Missing dependencies: libc6 (>= 2.4) [amd64], libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: libc6_2.39-0ubuntu8.2_amd64
Injecting package: libc6_2.39-0ubuntu8_amd64
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
Missing dependencies: libgcc-s1 [amd64], libidn2-0 (>= 2.0.5~) [amd64], libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: libgcc-s1_14-20240412-0ubuntu1_amd64
Injecting package: libidn2-0_2.3.7-2build1_amd64
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
Missing dependencies: libglib2.0-0 (>= 2.79.0) [amd64], gcc-14-base (= 14-20240412-0ubuntu1) [amd64], libunistring5 (>= 1.1) [amd64]
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: gcc-14-base_14-20240412-0ubuntu1_amd64
Injecting package: libunistring5_1.1-2build1_amd64
Missing dependencies: libglib2.0-0 (>= 2.79.0) [amd64]
Unsatisfied dependency: libglib2.0-0 (>= 2.79.0) [amd64]
gcc-14-base_14-20240412-0ubuntu1_amd64
libc6_2.39-0ubuntu8.2_amd64
libc6_2.39-0ubuntu8_amd64
libdconf1_0.40.0-4build2_amd64
libgcc-s1_14-20240412-0ubuntu1_amd64
libidn2-0_2.3.7-2build1_amd64
libunistring5_1.1-2build1_amd64
```

**Comment by neolynx** [comment]
Hi ! Thanks for reporting and the testing !

I will have a look into it...

**Comment by BrightGum** [comment]
Is there a way to increase the loglevel or the get more debug output?
I tried to set it by
```
  "logLevel": "debug",
  "logFormat": "default"
```
in /root/.aptly.conf as described in https://www.aptly.info/doc/configuration/ without success. 

How can I see the effective configuration of the command just executed?

**Comment by neolynx** [comment]
Interesting, a required dependency is not available in ubuntu: https://packages.ubuntu.com/noble/libdconf1

* [libc6](https://packages.ubuntu.com/noble/libc6) (>= 2.17) [arm64, ppc64el]
   -  GNU C Library: Shared libraries 
* [libc6](https://packages.ubuntu.com/noble/libc6) (>= 2.27) [riscv64]
* [libc6](https://packages.ubuntu.com/noble/libc6) (>= 2.4) [not arm64, ppc64el, riscv64]
* **libglib2.0-0 (>= 2.79.0) [amd64]
   -  Package not available** 
* [libglib2.0-0t64](https://packages.ubuntu.com/noble/libglib2.0-0t64) (>= 2.79.0) [not amd64]
   -  GLib library of C routines

**Comment by neolynx** [comment]
Same as in https://github.com/aptly-dev/aptly/pull/1317

**Comment by neolynx** [comment]
**Package: libdconf1**
Version: 0.40.0-4build2
Depends: libc6 (>= 2.4), libglib2.0-0 (>= 2.79.0)

**Package: libglib2.0-0t64**
Version: 2.80.0-6ubuntu3.1
Provides: libgio-2.0-0 (= 2.80.0-6ubuntu3.1), libglib-2.0-0 (= 2.80.0-6ubuntu3.1),                                                                   **libglib2.0-0 (= 2.80.0-6ubuntu3.1)**,

**Comment by neolynx** [comment]
the handling of provided packages has been fixed on master, please try latest CI build !

**Comment by BrightGum** [comment]
@neolynx 
I am currently on vacation, I will test it as soon as I am back at work, unless my colleagues are faster. 
And there is still the offer to support the project.

**Comment by neolynx** [comment]
Hi @BrightGum,

please enjoy your vacations !

and let me know how you would like to support the project, any help is welcome :) 
currently rebasing and revieweing pull requests and handling the the huge issue list are the main focus, in order to get the next release out soonish...

**Comment by BrightGum** [comment]
Hi @neolynx , I can confirm, that this issue is solved in version 1.5.0+207+g372ce3c4:
```
/root/go/bin/aptly version
aptly version: 1.5.0+207+g372ce3c4
```

There is no _Unsatisfied dependency_ anymore:
```
root@aptly-2404-v2:~/git/aptly# aptly snapshot  -dep-verbose-resolve  -with-deps  -dep-follow-all-variants  -dep-follow-recommends  search test-merge-2  'Name (= libdconf1)'
Missing dependencies: libc6 (>= 2.4) [amd64], libglib2.0-0 (>= 2.79.0) [amd64]
Injecting package: libc6_2.39-0ubuntu8.3_amd64
Injecting package: libc6_2.39-0ubuntu8_amd64
Injecting package: libglib2.0-0t64_2.80.0-6ubuntu3.1_amd64
Injecting package: libglib2.0-0t64_2.80.0-6ubuntu1_amd64
Missing dependencies: libgcc-s1 [amd64], libidn2-0 (>= 2.0.5~) [amd64], libffi8 (>= 3.4) [amd64], libmount1 (>= 2.26.2) [amd64], libpcre2-8-0 (>= 10.22) [amd64], libselinux1 (>= 3.1~) [amd64], zlib1g (>= 1:1.2.2) [amd64], libglib2.0-data [amd64], shared-mime-info [amd64], xdg-user-dirs [amd64]
Injecting package: libgcc-s1_14-20240412-0ubuntu1_amd64
Injecting package: libidn2-0_2.3.7-2build1_amd64
Injecting package: libffi8_3.4.6-1build1_amd64
Injecting package: libmount1_2.39.3-9ubuntu6_amd64
Injecting package: libpcre2-8-0_10.42-4ubuntu2_amd64
Injecting package: libselinux1_3.5-2ubuntu2_amd64
Injecting package: zlib1g_1:1.3.dfsg-3.1ubuntu2.1_amd64
Injecting package: zlib1g_1:1.3.dfsg-3.1ubuntu2_amd64
Injecting package: libglib2.0-data_2.80.0-6ubuntu3.1_all
Injecting package: libglib2.0-data_2.80.0-6ubuntu1_all
Injecting package: shared-mime-info_2.4-4_amd64
Injecting package: xdg-user-dirs_0.18-1build1_amd64
Missing dependencies: gcc-14-base (= 14-20240412-0ubuntu1) [amd64], libunistring5 (>= 1.1) [amd64], libblkid1 (>= 2.17.2) [amd64], libstdc++6 (>= 9) [amd64], libxml2 (>= 2.7.4) [amd64]
Injecting package: gcc-14-base_14-20240412-0ubuntu1_amd64
Injecting package: libunistring5_1.1-2build1_amd64
Injecting package: libblkid1_2.39.3-9ubuntu6_amd64
Injecting package: libstdc++6_14-20240412-0ubuntu1_amd64
Injecting package: libxml2_2.9.14+dfsg-1.3ubuntu3_amd64
Missing dependencies: libicu74 (>= 74.1-1~) [amd64], liblzma5 (>= 5.1.1alpha+20120614) [amd64]
Injecting package: libicu74_74.2-1ubuntu3.1_amd64
Injecting package: libicu74_74.2-1ubuntu3_amd64
Injecting package: liblzma5_5.6.1+really5.4.5-1build0.1_amd64
Injecting package: liblzma5_5.6.1+really5.4.5-1_amd64
Missing dependencies: 
gcc-14-base_14-20240412-0ubuntu1_amd64
libblkid1_2.39.3-9ubuntu6_amd64
libc6_2.39-0ubuntu8.3_amd64
libc6_2.39-0ubuntu8_amd64
libdconf1_0.40.0-4build2_amd64
libffi8_3.4.6-1build1_amd64
libgcc-s1_14-20240412-0ubuntu1_amd64
libglib2.0-0t64_2.80.0-6ubuntu3.1_amd64
libglib2.0-0t64_2.80.0-6ubuntu1_amd64
libglib2.0-data_2.80.0-6ubuntu3.1_all
libglib2.0-data_2.80.0-6ubuntu1_all
libicu74_74.2-1ubuntu3.1_amd64
libicu74_74.2-1ubuntu3_amd64
libidn2-0_2.3.7-2build1_amd64
liblzma5_5.6.1+really5.4.5-1build0.1_amd64
liblzma5_5.6.1+really5.4.5-1_amd64
libmount1_2.39.3-9ubuntu6_amd64
libpcre2-8-0_10.42-4ubuntu2_amd64
libselinux1_3.5-2ubuntu2_amd64
libstdc++6_14-20240412-0ubuntu1_amd64
libunistring5_1.1-2build1_amd64
libxml2_2.9.14+dfsg-1.3ubuntu3_amd64
shared-mime-info_2.4-4_amd64
xdg-user-dirs_0.18-1build1_amd64
zlib1g_1:1.3.dfsg-3.1ubuntu2.1_amd64
zlib1g_1:1.3.dfsg-3.1ubuntu2_amd64
```

Thank you for the support!
